### DISCUSSION TITLE
Glarnerdütsch

### DISCUSSION THREAD

**Discussion by Thahit** [Estheryu991/SwissGerman_Dictionary#15] [General]
Eifach w wiiteri art vo dialekt

**Comment by maku42** [comment]
Glarisdütsch: nüüt - nichts
Ich weiss nüüt - Ich weiss gar nicht.

Ich werde Glarnerdütsch übersetzeb uf Chinesisch, was bruche mer?
Ich werde Glarnerdütsch übersetzen auf Chinesisch, was brauchen wir?

Was sind die Unterregle von Natural Language Processing?
Was sind die Unterregle von Natural Language Processing gsi?
